sql/src/main/java/org/apache/druid/sql/avatica/DruidStatement.java

```diff
@@ -361,7 +364,7 @@ public void close()
             sqlLifecycle.finalizeStateAndEmitLogsAndMetrics(this.throwable, null, -1);
           }
         } else {
-          DruidMeta.logFailure(this.throwable);
+          errorHandler.logFailureAndSanitize(this.throwable);
```

Comment:
just log this here since no one is consuming the sanitized throwable? or have this be `this.throwable = errorHandler.logFailureAndSanitize(this.throwable`

Reply:
dont sanitize here